mh/database.py

```diff
@@ -0,0 +1,328 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Monster Hunter database module.
+
+    Monster Hunter 3 Server Project
+    Copyright (C) 2021  Sepalani
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+import random
+
+CHARSET = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+MEDIA_VERSIONS = {
+    "0903131810": "ROMJ08",
+    "0906222136": "RMHJ08",
+    "1001301045": "RMHE08",
+    "1002121503": "RMHP08",
+}
+
+
+def new_random_str(length=6):
+    return "".join(random.choice(CHARSET) for _ in range(length))
+
+
+class ServerType(object):
+    OPEN = 1
+    ROOKIE = 2
+    EXPERT = 3
+    RECRUITING = 4
+
+
+class LayerState(object):
+    JOINABLE = 0
+    EMPTY = 1
+    FULL = 2
+
+
+class City(object):
+    def __init__(self, name, parent):
+        self.name = name
+        self.parent = parent
+        self.capacity = 4
+        self.state = LayerState.EMPTY
+        self.players = set()
```

Comment:
What is your reasoning behind using `set` instead of `list`, to avoid duplicates more easily? I think the games have a few more packets with fields expecting a list index and with `set` I can't give them one because they are not reliable...

Reply:
I was using a `set` to avoid duplicates but if it's causing issues I can switch it to a list.

Reply:
Yeah, I think it would be better to switch to `list` in any case. If we manage to introduce duplicated values on to the list, it would be a bug to fix

Reply:
Alright, I'll amend this and update the PR, asap.